src/main/java/Problem1/LinkedListStack.java

```diff
@@ -3,32 +3,34 @@
 import java.util.LinkedList;
 
 public class LinkedListStack<T> implements Stack<T> {
-    // use Java LinkedList to store the data
-    // do not change member variables
     private LinkedList<T> data;
 
-    public LinkedListStack() {
-        // homework
+    public LinkedListStack()
+    {
+        this.data = new LinkedList<T>();
     }
 
     @Override
-    public boolean push(T val) {
-        // homework
-        return false;   // place holder
+    public boolean push(T val)
+    {
+        this.data.addLast(val);
```

Comment:
use First instead of Last. this was discussed at one of our hangout session too.